integration-tests/lvm/delete-thin.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/python
+
+# requirements: lvm volume group test with thin-pool thin-pool and thin thin1
```

Comment:
The comment seems to have replacement errors.

Reply:
It the type and the name.

Reply:
Ahh ok, I did not realize. Thanks